src/coffee/header.coffee

```diff
@@ -31,7 +31,7 @@ class Header
   #   y: 1
   #   z: 2
   toIndex: (name) ->
-    @h2i = _.object _.map @rawHeader, (head, index) -> [head, index] unless @h2i
+    if not @h2i then @h2i = _.object _.map @rawHeader, (head, index) -> [head, index]
```

Comment:
This was an old bug since it's creation five years ago. Basically, the `@/this` was bound to the `Global Object` in old JavaScript and became bound to `undefined` inside the arrow function. This fixes the problem by moving the condition to the outer scope instead.